### PR TITLE
python-dogpile.cache: update to 1.0.2.

### PR DESCRIPTION
**Pull request by cpixl**

### CODE REVIEW

**Comment by Chocimier** [comment]
Python2 version is not supported since 1.0.0, please [remove](https://github.com/void-linux/void-packages/commit/972d725701121f345dbe9d186f1d33bc3b717652), leaving only python3-dogpile.cache.

There is one package depending on it, but it can be removed too.

**Comment by cpixl** [comment]
Fixed and dropped `python-subliminal` too (`python3-subliminal` moved to `subliminal` now).

**Comment by Chocimier** [comment]
Good, please add transitional package from `python-subliminal` too and disable or fix do_check: https://github.com/void-linux/void-packages/pull/24513/files#diff-3460d28070cba26dfd72dfd4cb3e1501R21

**Comment by cpixl** [comment]
Fixed!

File: srcpkgs/subliminal/template [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	: # pypi tarball has no tests
+}
+
+post_install() {
+	vlicense LICENSE
+}
+
+python-subliminal_package() {
+	build_style=meta
+	depends="${pkgname}>=${version}_${revision}"
```
Comment (Chocimier) [inline srcpkgs/subliminal/template:31]: `${pkgname}` causes depenging on itself, just use `subliminal`.

File: srcpkgs/subliminal/template [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Template file for 'subliminal'
+pkgname=subliminal
+version=2.1.0
+revision=3
+archs=noarch
+wrksrc="subliminal-${version}"
```
Comment (Chocimier) [inline srcpkgs/subliminal/template:6]: wrksrc no more needed to set

File: srcpkgs/subliminal/template (hunk 1 above)
Comment (cpixl) [reply srcpkgs/subliminal/template]: fixed

File: srcpkgs/subliminal/template (hunk 2 above)
Comment (cpixl) [reply srcpkgs/subliminal/template]: fixed

**Pull request merged**